system/lib/libcxx/src/iostream.cpp

```diff
@@ -77,7 +77,7 @@ __asm__("?wclog@" _LIBCPP_ABI_NAMESPACE_STR "@std@@3V?$basic_ostream@_WU?$char_t
 #endif
 ;
 
-_LIBCPP_HIDDEN ios_base::Init __attribute__((init_priority(101))) __start_std_streams; // XXX EMSCRIPTEN: ensure a high priority for this constructor, see #3824
+_LIBCPP_HIDDEN ios_base::Init __start_std_streams _LIBCPP_INIT_PRIORITY_MAX;
```

Comment:
This change seems good, and looks like `test_iostream_ctors` should cover it...

Reply:
Yup, this is an example of something we upstreamed.

Reply:
Oh, cool! Open source ftw :smile: